src/Microsoft.HttpRepl.IntegrationTests/Commands/SetSwaggerCommandTests.cs

```diff
@@ -0,0 +1,324 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Moq;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class SetSwaggerCommandTests
+    {
+        [Fact]
+        public async Task ExecuteAsync_WithExactlyOneParseResultSection_WritesToConsoleManagerError()
+        {
+            MockedShellState shellState = new MockedShellState();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("section1");
+            HttpState httpState = new HttpState(null);
+
+            SetSwaggerCommand setSwaggerCommand = new SetSwaggerCommand();
+
+            await setSwaggerCommand.ExecuteAsync(shellState, httpState, parseResult, CancellationToken.None);
+
+            VerifyErrorMessageWasWrittenToConsoleManagerError(shellState);
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithExactlyTwoParseResultSections_SetsHttpStateSwaggerStructureToNull()
+        {
+            MockedShellState shellState = new MockedShellState();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("section1 sections2");
```

Comment:
Similar to the comment above, it feels odd to test this with the two sections of "section1" and "sections2" when the only way the program should get here is with "set" and "swagger" as the two sections. So I don't get too annoying with comments, this applies to all the tests here that make use of a ParseResult.

Reply:
Same as above comment.